utils/filters.py

```diff
@@ -23,7 +23,11 @@ def filters(self, msg, cmd, args, dry_run):
         return msg, cmd, args
 
     @cmdfilter
-    def filter_ignored_users(self, msg, cmd, args, dry_run):
-        if msg.frm.nick in self.bot_config.IGNORE_USERNAMES:
+    def filter_users(self, msg, cmd, args, dry_run):
+        name = msg.frm.fullname if BACKEND == 'Zulip' else msg.frm.nick
```

Comment:
this PR isnt about Zulip.  Please remove.  This isnt how we will do Zulip anyway; the patches for Zulip are public.